types/mocha/index.d.ts

```diff
@@ -388,36 +388,36 @@ declare namespace Mocha {
 
     // #region Test interface augmentations
 
-    interface HookFunction {
+    interface HookFunction<T = void> {
```

Comment:
Is `T` expected to be open-ended or are `Hook` and `void` the only valid types?

Reply:
right, that's a good point, let me add a constraint there